client/packages/electron/src/discovery/ServerDiscovery.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  useElectronClient,
+  InlineSpinner,
+  BaseButton,
+  AlertIcon,
+  Box,
+  Typography,
+  useTranslation,
+  frontEndHostDisplay,
+} from '@openmsupply-client/common';
+
+import { ServerDiscoveryLayout } from './ServerDiscoveryLayout';
+
+export const ServerDiscovery = () => {
+  const { servers, discoveryTimedOut } = useElectronClient(true);
+  const t = useTranslation('app');
+
+  return (
+    <ServerDiscoveryLayout
+      LoadingIndicator={
+        servers.length === 0 && <InlineSpinner color="secondary" showText />
+      }
+      ServerNodes={servers.map(server => {
+        const display = frontEndHostDisplay(server);
+
+        return (
+          <BaseButton
```

Comment:
Ideally we would remember last logged in server, and only present the list if we can't find it quickly (and maybe from login page allow you to go back to home page to connect to other server), but this should be sufficient for now ?

Reply:
could keep the previous server in localStorage easily enough - that's a neat idea.

Reply:
https://github.com/openmsupply/open-msupply/issues/58